### DISCUSSION TITLE
The license of the bundle is unclear

### DISCUSSION THREAD

**Discussion by AkihiroSuda** [crc-org/crc#3207] [General]
I checked this repo, the installer archive, https://cloud.redhat.com/openshift/install/crc/installer-provisioned , and https://code-ready.github.io/crc/ , but could not find the license of the bundle.

**Comment by gbraad** [comment]
* CRC itself is released under Apache-2.0
    * The bundle generation code lives in github.com/code-ready/snc and is similarly licensed
 * OCP and RHEL CoreOS are licensed under their conrresponding licenses and Terms and Conditions (you accepted these when downloading the bundle).

**Comment by AkihiroSuda** [comment]
>  you accepted these when downloading the bundle.

Thanks for the answer, but is it shown only on the first download?

**Comment by gbraad** [comment]
We will make sure this information is available, but at the moment we are still in pre-GA. This did not get our attention as they these concern projects and products we consume.

**Comment by praveenkumar** [comment]
> Thanks for the answer, but is it shown only on the first download?

Yes terms and conditions are shown only first time when you login/register but if you check https://www.openshift.com/legal/terms/ have most of info around it but as @gbraad said we still need to put our attention around those concerns.

**Comment by gbraad** [comment]
Note: for RHEL CoreOS not a single license applies, but each individual component is under their respective license as chosen by the upstream development team.

**Comment by gbraad** [comment]
Will ping legal again and PM
